src/Components/Scenes/Hydration/App/Transform/TransformEditor.tsx

```diff
@@ -105,12 +113,18 @@ const TransformEditor = (props: TransformProps) => {
       <div className="Editor__heading" />
       <div className={classes.name}>
         {isScriptNameActive ? (
-          <Field
+          <TextField
             autoFocus
-            onBlur={() => setIsScriptNameActive(!isScriptNameActive)}
+            onBlur={() => {
```

Comment:
This is working well for blur! However, it still doesn't seem to be saving on submit.

Reply:
It saves into Formik based on Blur.  On Submit will be an API call.

Reply:
Ahh ok makes sense